src/components/Autocomplete/components/ComboBox/tests/ComboBox.test.tsx

```diff
@@ -381,6 +381,60 @@ describe('<ComboBox/>', () => {
     });
   });
 
+  describe('onClose', () => {
+    it('gets called onBlur', () => {
+      const spy = jest.fn();
+      const comboBox = mountWithAppProvider(
+        <ComboBox
+          options={options}
+          selected={[]}
+          textField={renderTextField()}
+          onSelect={noop}
+          onClose={spy}
+        />,
+      );
+
+      comboBox.simulate('blur');
+      expect(spy).toHaveBeenCalled();
```

Comment:
We'll want to make sure it's only called once on blur

```suggestion
      expect(spy).toHaveBeenCalledTimes(1)
```